pages/curriculum.tsx

```diff
@@ -13,15 +13,20 @@ const Curriculum = () => {
 
   if (data) {
     const { lessons } = data
+    console.log(lessons)
+
     return (
       <Layout>
-        <Card title="Curriculum">
-          <ul>
-            {lessons.map((e: any) => (
-              <li key={e.id}>{e.title}</li>
-            ))}
-          </ul>
-        </Card>
+        {lessons
```

Comment:
please don't put logic inside the rendering component. Create a variable like `lessonList` before `return` and render variable here. This helps keep the component code clean.